cdap-docs/admin-manual/source/upgrading/cloudera.rst

```diff
@@ -44,6 +44,10 @@ CDAP:
 
 #. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading *authorization* enabled CDAP instance you will need to give *cdap* user *ADMIN* privilege on
+   all existing CDAP Namespaces. Note: *ADMIN* privilege does not give *cdap* user any access (*READ/WRITE*)
+   privileges on the Namespaces.
```

Comment:
You may also want to mention that this is a temporary grant required only for the purpose of running the upgrade tool. Once upgrade completes, these grants can be revoked.